tests/system/action/base.py

```diff
@@ -48,24 +56,22 @@ def request_multi(
                 }
             ],
             anonymous=anonymous,
+            lang=lang,
         )
         if response.status_code == 200:
             results = response.json.get("results", [])
             assert len(results) == 1
             assert results[0] is None or len(results[0]) == len(data)
         return response
 
-    def request_json(self, payload: Payload, anonymous: bool = False) -> Response:
-        client = self.client if not anonymous else self.anon_client
-        return client.post(ACTION_URL, json=payload)
-
-    def request_lang(
-        self, action: str, data: Dict[str, Any], lang: str = "en_US"
+    def request_json(
+        self, payload: Payload, anonymous: bool = False, lang: Optional[str] = None
     ) -> Response:
-        payload = [{"action": action, "data": [data]}]
-        headers = {"Accept_Language": lang}
-
-        return self.client.post(ACTION_URL, json=payload, headers=headers)
+        client = self.client if not anonymous else self.anon_client
+        headers = {}
+        if lang:
+            headers["Accept_Language"] = lang
```

Comment:
```suggestion
            headers["Accept-Language"] = lang
```
I'm wondering why that even worked...

Reply:
Changed.